tendrl/node_agent/manager/manager.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, machine_id):
         ).__init__(
             "node",
             utils.get_local_node_context(),
+            utils.get_local_node_context(),
```

Comment:
@shtripat i feel instead of doing multiple file reading, read once and store that in var and reuse the var twice. is that ok?